src/tests/test_magic.py

```diff
@@ -196,11 +196,14 @@ def test_persist_bare(ip):
     assert result.error_in_exec
 
 
-def get_table_rows_as_dataframe(ip, table_name):
-    save_df_name = f"df_{table_name}"
+def get_table_rows_as_dataframe(ip, table_name, desire_saved_df_name=None):
```

Comment:
Let's add a short description to this function.
I think we can rename `table_name` -> `table` and `desire_saved_df_name` -> `name`
I also think we can add an optional parameter `limit = 1`.

Now, this function returns a single row from a specified table as a dataframe, with the dataframe named after the table. Then, you can change the number of rows and the name of the dataframe.